VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Public/RuleAttributes.h

```diff
@@ -126,6 +126,9 @@ class VITRUVIO_API URuleAttribute : public UObject
 	UPROPERTY()
 	bool Hidden;
 
+	UPROPERTY()
+	bool UserSet;
```

Comment:
I think this attributes should start with b

Reply:
True! Also fixed "Hidden".